Source/Platform/Assert.h

```diff
@@ -77,12 +77,16 @@ namespace Helium
 		/// @name Static Utility Functions
 		//@{
 		static bool Trigger( const char* pExpression, const char* pFunction, const char* pFile, int line, const char* pMessage, ... );
+		static void SetAssertIsFatal(bool isFatal) { sm_assertIsFatal = isFatal; }
```

Comment:
put the implementation of this in a cpp (typically i would say inl file, but this isn't high frequency enough to bother)